src/components/SetupKeyEdit.tsx

```diff
@@ -0,0 +1,496 @@
+
+          <Row style={{ marginTop: "30px" }}>
+            <Container style={{ width: "100%", padding: 0, margin: 0 }}>
+              <Row>
+                <Col span={4}>
+                  <Paragraph style={{ margin: 0, fontWeight: "bold" }}>
+                    Expires
+                  </Paragraph>
+                  <Row>
+                    <Input
+                      style={{ marginTop: "5px" }}
+                      disabled
+                      suffix={<LockOutlined style={{ color: "#BFBFBF" }} />}
+                      value={
+                        customExpiresFormat(new Date(formSetupKey.expires))!
+                      }
+                    />
+                  </Row>
+                </Col>
+              </Row>
+            </Container>
+          </Row>
+
+          <Row style={{ marginTop: "30px" }}>
+            <Col span={24}>
+              <Paragraph
+                style={{
+                  whiteSpace: "pre-line",
+                  margin: 0,
+                  fontWeight: "bold",
+                }}
+              >
+                Auto-assigned groups
+              </Paragraph>
+            </Col>
+            <Col span={6}>
+              <Form.Item
+                style={{ marginTop: "5px", marginBottom: 0 }}
+                name="autoGroupNames"
+                rules={[{ validator: selectValidator }]}
+              >
+                <Select
+                  mode="tags"
+                  style={{ width: "100%" }}
+                  placeholder="Associate groups with the key"
+                  tagRender={tagRender}
+                  dropdownRender={dropDownRender}
+                  // enabled only when we have a new key !setupkey.id or when the key is valid
+                  disabled={!(!setupKey.id || setupKey.valid)}
+                >
+                  {tagGroups.map((m) => (
+                    <Option key={m}>{optionRender(m)}</Option>
+                  ))}
+                </Select>
+              </Form.Item>
+            </Col>
+          </Row>
+          <Row style={{ marginTop: "40px", marginBottom: "28px" }}>
+            <Text type={"secondary"}>
+              Learn more about
+              <a
+                target="_blank"
+                rel="noreferrer"
+                href="https://docs.netbird.io/how-to/register-machines-using-setup-keys"
+              >
+                {" "}
+                setup keys
+              </a>
+            </Text>
+          </Row>
+        </Form>
+        <Container
+          style={{
+            display: "flex",
+            flexDirection: "row",
+            justifyContent: "start",
+            padding: 0,
+            gap: "10px",
+          }}
+          key={0}
+        >
+          <Button onClick={onCancel}>Cancel</Button>
+          <Button
+            type="primary"
+            style={{
+              height: "100%",
+              fontSize: "14px",
+              borderRadius: "2px",
+            }}
```

Comment:
```suggestion
```

Reply:
no need for these styles, let's do similar to other pages (see the UserEdit component)

Reply:
done